src/mon/MgrMap.h

```diff
@@ -448,7 +451,7 @@ class MgrMap
       ENCODE_FINISH(bl);
       return;
     }
-    ENCODE_START(13, 6, bl);
+    ENCODE_START(14, 6, bl);
```

Comment:
ACK for this line and appending `force_disabled_modules` at the very end.
However, I don't the corresponding bump up of version in the decoder.